docs/debugging.rst

```diff
@@ -0,0 +1,230 @@
+Debugging
+=========
+
+Debugging a massive-parallelized CPU/GPU program is not easy, so Taichi provides some
+builtin utilities that hopefully could help you debug your Taichi program.
+
+Runtime ``print`` in kernel
```

Comment:
How about change ``Runtime`` to `Run-time`? Maybe ``Runtime`` are usually referred to some environment setting such as ``JVM`` or ``MSVC runtime``, and ``Run-time`` can be more precisely to say the execution of our program, and our titles could be consisitent with each other(``Run-time`` and ``Compile-time``)
```suggestion
Run-time ``print`` in kernels
```